doc/schemas/listfunds.schema.json

```diff
@@ -186,6 +187,11 @@
               "DUALOPEND_AWAITING_LOCKIN"
             ],
             "description": "the channel state, in particular \"CHANNELD_NORMAL\" means the channel can be used normally"
+          },
+          "channel_id": {
+            "type": "hash",
+            "description": "The full channel_id (funding txid Xored with output number)",
+            "added": "v23.02"
```

Comment:
Sorry, this should be v23.05 now, since v23.02 has already been released!

Reply:
Yes, this was proposed before v23.02 release.